.github/workflows/main.yml

```diff
@@ -55,12 +55,13 @@ jobs:
           - python: "3.7"
             image_tag: "3.20"
           - python: "3.8"
+            pulp_cli: "0.16.0"
             image_tag: "3.19"
           - python: "3.9"
             image_tag: "3.18"
```

Comment:
Remind me again what `pulp_cli` defaults to if we do not explicitly set it?

Reply:
"from source" (the current main branch)